run-tests

```diff
@@ -1,5 +1,7 @@
-#!/bin/sh
+#!/bin/sh --
 
+set -eu
+sudo dnf -y install lvm2 python3-inotify python3-sphinx python3-docutils python3-PyYAML python3-jinja2 python3-lxml python3-pylint python3-coverage
```

Comment:
Can we add it to README instead, or make it conditional in some way? `dnf` likes to refresh metadata which takes time and having that every time in the middle of debugging will be annoying.

Reply:
Fixed by checking with `rpm -q --quiet` first.